Apps/WebClient/src/NewClientApp/src/constants/entryType.ts

```diff
@@ -159,4 +160,17 @@ entryTypeMap.set(EntryType.DiagnosticImaging, {
     reportEventName: "Diagnostic Imaging Exams",
 });
 
+entryTypeMap.set(EntryType.BcCancerScreening, {
+    type: EntryType.BcCancerScreening,
+    commentType: CommentEntryType.BcCancerScreening,
+    name: "BC Cancer Results",
+    description:
+        "View and download your results as soon as they are available.",
+    icon: "ribbon",
+    component: "BcCancerScreeningResultTimelineComponent",
+    eventName: "bc_cancer_screening",
+    moduleName: "BcCancerScreening",
+    reportEventName: "BC Cancer Screening Exams",
```

Comment:
I don't think we have a report for this yet, but I'd suggest removing "Exams" from the name unless it's been requested.